server/index.js

```diff
@@ -73,7 +73,39 @@ app.get('/images/:imageId', (req, res, next) => {
       if (!result.rows[0]) {
         throw new ClientError(404, `cannot find product with imageId ${imageId}`);
       }
-      res.json(result.rows[0]);
+      res.status(200).json(result.rows[0]);
+    })
+    .catch(error => next(error));
+});
+
+app.post('/images/likedImage', (req, res, next) => {
+  const { imageId, userId } = req.body;
+  if (!imageId || !userId) {
+    throw new ClientError(400, 'imageId and userId must be positive integers');
+  }
+  const sql = `
+    insert into "Liked_Image" ("imageId", "userId")
+         select "imageId", "userId"
+           from "Images"
+          where "imageId" = $1 and "userId" = $2
```

Comment:
If you already know the imageId and userId, why do you need to select them again from Images?